sdk/boms/azure-core-dependencies/README.md

```diff
@@ -0,0 +1,51 @@
+# Azure external dependencies BOM for libraries and services
+The Azure external dependencies BOM for libraries provides all shared artifacts that are used in Java development for
+Azure. It provides a simple and elegant way to manage dependencies for multiple Azure libraries and services.
+
+## Table of contents
+- [Getting started](#getting-started)
+  - [Adding the BOM to your project](#adding-the-bom-to-your-project)
+  - [Adding libraries to your project](#adding-libraries-to-your-project)
+
+## Getting started
+
+### Adding the BOM to your project
+
+To consume the BOM include it in the `dependencyManagement` section of your project's POM. Note that this **does not**
+result in all dependencies being included in your project.
+
+```xml
+<dependencyManagement>
+  <dependencies>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-external-dependencies-bom</artifactId>
```

Comment:
Should the name be updated to `azure-core-dependencies-bom`?

Reply:
Yes, if this is being kept this should be renamed to `azure-core-dependencies`.